tools/ResourcesExtractor/ResourcesExtractor/Magic.cs

```diff
@@ -96,22 +97,23 @@ public enum Languages
 		zh_TW = 0x0404,
 	}
 
-	public static unsafe string GetLocalizedResource(int resourceId, int langid)
+	public static unsafe string GetLocalizedResource(int resourceId, int? langid = null, LANG language = LANG.LANG_ENGLISH, SUBLANG subLanguage = SUBLANG.SUBLANG_ENGLISH_US)
```

Comment:
I don't think we should change the signature here. To me, the only needed change would be in `GetResources` method (in MainWindow.xaml.cs) so that it includes resources from 5114 to 5155

Reply:
This is what I have in mind to be added in `GetResources`:

```
		for (int i = 5114; i <= 5155; i++)
		{
			allResources.Add((i.ToString(CultureInfo.InvariantCulture), i));
		}
```

Reply:
Thanks @Youssef1313! I would like to also keep the FilePicker opening, as it is convenient when you don't have to search for the files.

Reply:
Yeah keeping that is good. Though the signature of this method should still not change

Reply:
Probably nothing in this whole file should change.

Reply:
Thanks. I didn't commit the changes in the second file 🙃

Reply:
Jup, seems to work well without the changes as well.

Reply:
Great. Thanks!